src/redux/configureStore.js

```diff
@@ -0,0 +1,11 @@
+import { configureStore, combineReducers } from '@reduxjs/toolkit';
+import categoryReducer from './categories/Categories';
+import bookReducer from './books/Books';
+
+const rootReducer = combineReducers({
+  book: bookReducer,
+  category: categoryReducer,
+});
+const store = configureStore({ reducer: rootReducer });
```

Comment:
- [OPTIONAL] 🤔 What If you Import `applyMiddleware`  to use some handy Redux middleware, like a redux logger.  it will help you a lot in the development process. 👍🏻 To install it, you need to run: 

  ```  npm i --save redux-logger  ```

Reply:
Thanks I'll apply that one :)